.github/workflows/validate-gentx.yml

```diff
@@ -0,0 +1,94 @@
+name: Validate Gentx
+on:
+  pull_request:
+
+jobs:
+  find-gentx-files:
+    runs-on: ubuntu-latest
+    outputs:
+      gentx-files: ${{ steps.set-outputs.outputs.gentx-files }}
+      gentx-changed: ${{ steps.set-outputs.outputs.gentx-changed }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Get changed gentx files
+        id: changed-files
+        uses: tj-actions/changed-files@v39
+        with:
+          files: "./celestia/*gentx*/*.json"
+      - name: Set outputs
+        id: set-outputs
+        run: |
+          echo "gentx-files=${{ steps.changed-files.outputs.all_changed_files }}" >> "$GITHUB_OUTPUT"
+          echo "gentx-changed=${{ steps.changed-files.outputs.any_changed }}" >> "$GITHUB_OUTPUT"
+
+  check-user-pr:
+    needs: find-gentx-files
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: needs.find-gentx-files.outputs.gentx-changed == 'true'
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+      - name: Check user csv
+        id: check-user-pr
+        run: go run .github/workflows/checkuser.go ${{github.actor}} ${{github.event.number}}
+      - name: Commit changes to the csv
+        if: success()
+        uses: EndBug/add-and-commit@v9
+        with:
+          message: "Add ${{github.actor}} with pr ${{github.event.number}} to usersprs.csv file"
+      - name: Add comment about user in csv
+        if: failure()
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'User has previously submitted a gentx PR'
```

Comment:
There seems sth off with this. I think we are hitting 
https://github.com/celestiaorg/networks/blob/14fb188af6e5881b1f3d52786c64cfe70643570d/.github/workflows/checkuser.go#L49C18-L49C18 

in https://github.com/celestiaorg/networks/actions/runs/6597322356/job/17924245542?pr=362

but it's the only gentx they submitted 🤔